cmake/thirdparty/get_nvtx.cmake

```diff
@@ -0,0 +1,27 @@
+# =============================================================================
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+# This function finds NVTX and sets any additional necessary environment variables.
+function(find_and_configure_nvtx)
```

Comment:
We should probably set up NVTX in rapids-cmake since we have three repos using the same code (rmm, cudf, cuspatial -- can't recall if we finished cuspatial but it's supposed to use this).

Reply:
I agree, I was surprised to find it wasn't already in rapids-cmake.

Reply:
https://github.com/rapidsai/rapids-cmake/pull/606